.travis.yml

```diff
@@ -91,15 +91,16 @@ script:
 - if [[ "$RDKIT" == true && "$OPENEYE" == true ]];
   then
     pytest --ignore=utilities --ignore=examples/deprecated --ignore=docs
-    --ignore=devtools --doctest-modules --nbval-lax --cov=openforcefield;
+    --ignore=devtools --doctest-modules --nbval-lax --cov=openforcefield
+    --cov-config=setup.cfg;
   elif [[ "$OPENEYE" == true ]];
   then
     pytest --ignore=utilities --ignore=examples/deprecated --ignore=docs
     --ignore=devtools --ignore=examples/check_dataset_parameter_coverage
-    --nbval-lax --cov=openforcefield;
+    --nbval-lax --cov=openforcefield --cov-config=setup.cfg;
```

Comment:
Pinging @dgasmith @janash @Lnaden to be aware of this issue.

Basically, I removed our old coverage version pin, and suddenly codecov dropped by 30%. This was due to the code lines _in the tests themselves_ becoming a part of the denominator for our coverage % (but oddly, not the numerator).

Per the advice [here](https://pytest-cov.readthedocs.io/en/latest/config.html), this can be avoided using the `[omit]` keyword in a config file. This is present both in the OFFTK repo and in the cookiecutter, in [`setup.cfg`](https://github.com/openforcefield/openforcefield/blob/master/setup.cfg#L3-L9). However, our `pytest` commands (and the cookiecutter's) don't specifically point to this file (`--cov-config`). 

I suspect that new versions of `coverage` or `pytest-cov` have changed the way that config files are found, such that `setup.cfg` is no longer found by default. This delta should fix that if it's also an issue on your end.

Reply:
Thats odd, you shouldn't need to pass this in unless you are running pytest in a different folder.

Reply:
The docs are confusing on this point. In fact, they seem to say two opposite things about default behavior in adjacent sentences.

> Caveats
> =======
> 
> A unfortunate consequence of coverage.py's history is that ``.coveragerc`` is a magic name: it's the default file but it also
> means "try to also lookup coverage configuration in ``tox.ini`` or ``setup.cfg``".
> 
> In practical terms this means that if you have your coverage configuration in ``tox.ini`` or ``setup.cfg`` it is paramount
> that you also use ``--cov-config=tox.ini`` or ``--cov-config=setup.cfg``.